pages/donate.js

```diff
@@ -14,41 +14,54 @@ class Donate extends Component {
   render () {
     return (
       <PageWrapper title='Donate – The Teacher Fund'>
-        <div className='w-100 h-100 flex'>
-          <div className='w-100 h-100 absolute bg-tf-gray o-10 z-minus-1' />
-          {process.browser && <div className='flex flex-column m-auto justify-between pa3 pa4-ns'>
-            <div className='flex flex-column pa4-ns pa2 pb4 tf-lato tc mv-auto'>
-              <h1 className='tf-dark-gray f1 tf-oswald fl pb3 mv0'>
-                Fund Teachers. Help Students.
+        {process.browser &&
+        <React.Fragment>
+          <div className='flex flex-column bg-trans-gray justify-between ph3 pv4 pv5-ns pa4-ns'>
+            <div className='flex flex-column tf-lato tc mv-auto'>
+              <h1 className='tf-dark-gray f2 f1-l tf-oswald fl mv0'>
```

Comment:
Since the padding from the bottom was removed from the `h1`, can you remove `mv0`? I added that originally because of the padding class that someone had added, but now there's not any breathing room between the heading and the text below.

Reply:
removed